constants.js

```diff
@@ -455,6 +455,13 @@ const Errors = {
 			message: 'LSP14: Cannot transfer ownership to address(this)',
 		},
 	},
+	LSP17ContractExtension: {
+		'0x3d746f74': {
+			error: 'NoExtensionForFunctionSelector(bytes4)',
```

Comment:
maybe we could rename it to?

```suggestion
			error: 'NoExtensionFoundForFunctionSelector(bytes4)',
```